docs/Oracle Jobs/adapters.md

```diff
@@ -9,12 +9,12 @@ Core adapters are the built-in functionality that every Chainlink node supports.
 
 Adapters that are prefixed with "Eth" refer to tasks that post data onto the chain. Here are some examples of the data types that adapters convert data to.
 
-| Name              | Core Adapter    | Ethereum Data Type |
-|-------------------|------------|--------------------|
-| Signed Integers   | [EthInt256](#ethint256)  | int256             |
+| Name              | Core Adapter              | Ethereum Data Type |
+| ----------------- | ------------------------- | ------------------ |
+| Signed Integers   | [EthInt256](#ethint256)   | int256             |
 | Unsigned Integers | [EthUint256](#ethuint256) | uint256            |
 | Bytes             | [EthBytes32](#ethbytes32) | bytes32            |
-| Boolean           | [EthBool](#ethbool)    | bool               |
+| Boolean           | [EthBool](#ethbool)       | bool               |
 
 You can learn more about Solidity data types <a href="https://docs.soliditylang.org/en/v0.5.3/types.html" target="_blank">here</a>.
```

Comment:
Replace <a> with a markdown version

Reply:
This PR isn't to clean up old code. Can we focus on the changes I'm making?